src/pages/index.tsx

```diff
@@ -1,29 +1,48 @@
 import CampaignHero from "../components/hero/campaignHero";
-import { featuredImageID } from "../utils/utils";
-import {
-  SkipNavigation,
-  TemplateAppContainer,
-} from "@nypl/design-system-react-components";
+import SwimLanes from "../components/swimlanes/swimLanes";
+import { TemplateAppContainer } from "@nypl/design-system-react-components";
+import data from "@/data/lanes";
+import { getNumItems, featuredImageID } from "@/utils/utils";
 
 export default function Home(props: any) {
   return (
     <TemplateAppContainer
       aboveHeader={<p> Notification banner </p>}
       header={<p> Header </p>}
       breakout={<CampaignHero featuredImageID={props.featuredImageID} />}
-      contentPrimary={
-        <>
-          <p>First swim lane</p>
-          <p>Featured Content</p>
-          <p>Rest of swim lanes</p>
-          <p>Explore further links</p>
-        </>
-      }
+      contentPrimary={<SwimLanes lanesWithNumItems={props.lanesWithNumItems} />}
       renderSkipNavigation={true}
     />
   );
 }
 
 export async function getServerSideProps() {
-  return { props: { featuredImageID: featuredImageID() } };
+  const lanes = data.lanes;
+  const flatCollections = [].concat(...lanes.map((lane) => lane.collections));
+  const collectionsWithNumItems = await Promise.allSettled(
```

Comment:
This and the `result.status === "fulfilled"` is good. We can refactor it a bit more to add error handling but let's make that a refactor ticket to work on later.